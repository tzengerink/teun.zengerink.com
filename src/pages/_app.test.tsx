import { render } from '@testing-library/react'
import { Router } from 'next/router'
import React from 'react'
import App from './_app.page'

describe('App', () => {
  const Component: React.FC = () => <div>[COMPONENT]</div>
  const router = { events: { on: jest.fn(), off: jest.fn() } } as unknown as Router

  it('renders correctly', () => {
    const { container } = render(<App Component={Component} router={router} pageProps={{}} />)
    expect(container).toMatchSnapshot()
  })
})
