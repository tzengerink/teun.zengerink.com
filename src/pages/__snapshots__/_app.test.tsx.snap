// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`App renders correctly 1`] = `
<div>
  <div>
    [COMPONENT]
  </div>
</div>
`;
